src/pages/instant-observability.js

```diff
@@ -722,6 +696,14 @@ const QuickstartsPage = ({ data, location }) => {
                 css={css`
                   ${view === VIEWS.GRID && `height: 100%;`}
                   background-color: var(--tertiary-background-color);
+
+                  @media screen and (max-width: ${COLLAPSE_BREAKPOINT}) {
+                    flex-direction: row;
+
+                    img {
```

Comment:
This works for most tiles, but some tiles that do not have an image use a `div` instead. I would recommend moving this breakpoint logic to the `PackTile` component itself, that way it's all encapsulated and you can deal with the different types of display "images".